### ISSUE TITLE
get_point() in class View Point: point_view variable not updating tkinter view_point_label

### ISSUE THREAD

**Issue by chetjones003** [chetjones003/COGO-Solver#3]
```python
class ViewPoint(tk.Frame):

        self.get_point_number_var = tk.IntVar(self)
        self.point_view = tk.StringVar()

        submit_point_button = ttk.Button(self, text = 'Submit Point Number', padding=(20,5,20,5), command = lambda : self.get_point(self.get_point_number_var))
        submit_point_button.grid(column=2, row=4, pady=10)

        view_point_label = ttk.Label(self, textvariable = self.point_view, font = 'Calibri 15')
        view_point_label.grid(column=1, row=3, columnspan=2, pady=10) 

    def get_point(self, p):
        p = self.get_point_number_var.get()
        print(p)
        self.old_point = Point(p, 0, 0, 0)
        point = self.old_point.view_point()
        print(point)
        self.point_view = str(point)
```

Attribute `textvariable` in `view_point_label` will not show the string returned from `get_point()`
If I print the point inside the `get_point` function the point is printed properly in the console but will not update inside the gui.

#### If Applicable
Code from the Point class
```python
from tkinter import filedialog

class Point:
    storage = []
    point_info = []
    N = 0
    E = 0
    Z = 0

    def __init__(self, p, n, e, z):
        self.p = p
        self.n = n
        self.e = e
        self.z = z

    def list_conversion(self, point_file):
        with open(point_file, 'rt') as point_file:
            for point in point_file:
                # Break down the stored string into a string separated by whitespace
                point = point.replace('[', '')
                point = point.replace(']', '')
                point = point.replace(',', '')

                # turn the string into integers in one list
                point = [int(i) for i in point.split()]

                # turn into a list of lists
                self.storage = [point[i:i+4] for i in range(0, len(point), 4)]
                return self.storage

    def store_points(self):
        point_file = filedialog.askopenfilename()
        with open(point_file, 'at') as point_file:
            self.storage.append([self.p, self.n, self.e, self.z])
            print(self.storage)
            point_file.write(str(self.storage) + ", ")

    def view_point(self):
        point_file = filedialog.askopenfilename()
        self.list_conversion(point_file)
        # get point from input point number
        for point_info in self.storage:
            if point_info[0] == self.p:
                return point_info

    def get_point(self):
        point_file = filedialog.askopenfilename()
        self.list_conversion(point_file)
        # get point from input point number then separate and store into separate values for later use
        for point_info in self.storage:
            if point_info[0] == self.p:
                self.n = point_info[1]
                self.e = point_info[2]
                self.z = point_info[3]

```

**Comment by chetjones003** [comment]
Issue solved with [Pull Request #4](https://github.com/chetjones003/COGO-Solver/pull/4)

**Comment by chetjones003** [comment]
Also point_view.set(str(point))
